chain/indexer.go

```diff
@@ -158,10 +160,13 @@ func NewTipSetIndexer(node lens.API, d model.Storage, window time.Duration, name
 
 // TipSet is called when a new tipset has been discovered
 func (t *TipSetIndexer) TipSet(ctx context.Context, ts *types.TipSet) error {
-	ctx, span := otel.Tracer("").Start(ctx, "Indexer.TipSet")
-	if span.IsRecording() {
-		span.SetAttributes(attribute.String("tipset", ts.String()), attribute.Int64("height", int64(ts.Height())))
-	}
+	ctx, span := otel.Tracer("").Start(ctx, "TipSetIndexer.TipSet", trace.WithAttributes(
```

Comment:
Can we optimize this to avoid allocations for the attributes? `ts.String()` allocates for example. The original code used `span.IsRecording` to do this. We should aim to keep tracing impact very low when it is disabled.

Reply:
done.